src/test/ui/issue-23302-2.stderr

```diff
@@ -0,0 +1,15 @@
+error[E0391]: unsupported cyclic reference between types/traits detected
```

Comment:
seems like our message should include `consts`. Or maybe just change altogether. Something like "unsupported cyclic dependency detected"?

Reply:
Sounds like a good idea

Reply:
It's about time. Maybe not even "unsupported", that always seemed a bit unnecessary.